Sources/SourceKitLSP/Swift/MacroExpansion.swift

```diff
@@ -69,6 +73,9 @@ extension SwiftLanguageService {
 
     let expansion = try await self.refactoring(expandMacroCommand)
 
+    var completeExpansionFileContent = ""
+    var completeExpansionFilePath = self.generatedMacroExpansionsPath
+    var macroExpansionFilePaths = [URL]()
```

Comment:
Nitpick, the preferred spelling is

```suggestion
    var macroExpansionFilePaths: [URL] = []
```